common/lib/xmodule/xmodule/course_metadata_utils.py

```diff
@@ -61,10 +61,26 @@ def display_name_with_default(course):
             said course.
     """
     # TODO: Consider changing this to use something like xml.sax.saxutils.escape
+    return course.display_name_unescaped.replace('<', '&lt;').replace('>', '&gt;')
```

Comment:
@robrap, I agree.  Please note that the "TODO" line at this point in the code was already there.  It suggests that the original author was also a bit concerned that escaping only the < and > characters is insufficient.

So,  how would you like to proceed? --
1. replace this code with better code that escapes completely?
2. leave the escaping to other parts of the code (e.g. the templates)?
3. something else?